css/components/style.css

```diff
@@ -18,3 +16,142 @@ body {
     font-weight: 300;
     font-style: normal;
 }
+
+#section-1 {
+  height: 670px;
+  position: relative;
+}
+
+.mw-background {
+  background: url('/assets/images/intro.jpg') no-repeat center center fixed;
+    background-size: 100%;
+    width:100%;
+    height: 100%;
+    opacity: 0.6;
+}
+
+.mw-header {
+  position: absolute;
+    top: 50%;
+    left: 15%;
+    font-size: 100px;
+}
+
+.mw-container-1 {
+  padding: 150px 200px 0;
+  height: 500px;
+  background-color: white;
+}
+
+.mw-container-1 > .mw-container-left {
+  height: 250px;
+  width: calc(50% - 20px);
+  display: inline-block;
+  background-image: url('/assets/images/section2.jpg') ;
```

Comment:
la asta nu putea-i sa pui in containerul cu clasa .mw-container-left un **img** , e mai usor sa manipulezi imaginile asa
`
<div class="mw-container-left"> 
<img src="/assets/images/section2.jpg">
</div>
`

ceva de genul